templates/public/donate.html

```diff
@@ -53,6 +53,15 @@
     <img width="496" height="76" src="{% static "pia_logo.png" %}"
     class="sponsor-btn-pia" title="" alt="Private Internet Access logo"/></a>
 
+    <p>We would also like to thank <a href="https://icons8.com/" title="Icons8">
+    Icons8</a> for their yearly monetary donation. Icons8 is a team of talented
+    designers and engineers working on fullfilling all your icons needs.</p>
```

Comment:
fulfilling*

Reply:
There is no such thing as too many l's!

Thanks, will fix it.

Reply:
Done.